scripts/ci-build.sh

```diff
@@ -0,0 +1,562 @@
+#!/bin/bash -e
+
+# (C) 2018 GENIVI Alliance
+# LICENSE: MPLv2
+
+# ---- General settings ----
+
+# Fixed cache location on all agents that support this.
+AGENT_STANDARD_DL_DIR="/var/cache/yocto/downloads"
+AGENT_STANDARD_SSTATE_DIR="/var/cache/yocto/sstate"
+AGENT_STANDARD_SGX_LOCATION="/var/go/sgx_bin"
+AGENT_STANDARD_SGX_GEN3_LOCATION="/var/go/sgx_bin_gen3"
```

Comment:
All of these paths are specific to the CI environment on go.genivi.org. I'd prefer them to be either be tied to $HOME or $PWD if possible and then change them locally in the CI environment.

Reply:
These have the prefix AGENT_ because they are for the build agents.  They affect the user build only depending on REUSE_*, and if the user does not set its own SGX_DRIVERS location, etc.
So I propose to keep them.  We could set a user-friendly default, but what would that be?
Proposal?  $HOME/something?